test/backingstore/rev-backingstore.py

```diff
@@ -0,0 +1,100 @@
+#
```

Comment:
Does it need a separate `.py` file considering the new, more universal `rev-model-options-config.py`?   It looks complicated and specific for this test, but that would be a question I'd ask for every new test.

Reply:
for this test, its fairly vital to maintain a nonstandard config file